src/botPage/view/react-components/NotoficationBanner.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { translate } from '../../../common/i18n';
+import { isEuCountry, isUKCountry } from '../../../common/footer-checks';
+import { getTokenList, get as getStorage } from '../../../common/utils/storageManager';
+import { AppConstants } from '../../../common/appId';
+
+const NotificationBanner = ({ api }) => {
+    const [showBanner, setShowBanner] = React.useState(false);
+    const tokenList = getTokenList();
+    const landingCompanyName = [];
+
+    React.useEffect(() => {
+        checkForShowBanner(landingCompanyName);
+    }, []);
+
+    const getActiveToken = activeToken => {
+        const activeTokenObject = tokenList.filter(tokenObject => tokenObject.token === activeToken);
+        return activeTokenObject.length ? activeTokenObject[0] : tokenList[0];
+    };
+
+    const checkForShowBanner = () => {
+        if (tokenList.length) {
```

Comment:
we use if/else It's more clear if we check the else before our long if statement 

```suggestion
        if (!tokenList.length) {
            isEuUK(api, true);
            return
        }
```